dnanexus/src/eggd_athena.sh

```
@@ -0,0 +1,86 @@
+
+    chmod a+x bedtools
+    sudo mv bedtools /usr/local/bin
+
+    echo "Installing python packages"
+    time sudo -H python3 -m pip install --no-index --no-deps packages/*
+
+    _set_bool_inputs
+    _set_string_inputs
+
+    python3 athena/athena.py \
+        --regions "$regions_path" \
```

Shellcheck (0.10.0)

[warning] regions_path is referenced but not assigned.

(SC2154)